File: Assignment_16/program4.c
/////////////////////////////////////////////////////////////////
//
// Name :          program4.c
// Discription :   Display Pattern 
// Input :         Integer
// Output :        Charecter
// Author :        Shivam Rajendra Kale
// Date :          22/05/2025
//
/////////////////////////////////////////////////////////////////

/*
    Input : row = 6  col = 6
    
    Output : *  *  *  *  *  *
             *  *           *
             *     *        *
             *        *     *
             *           *  *
             *  *  *  *  *  *
*/

#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0, j = 0;

    if(iRow != iCol)
    {
        return;
    }

    for(i = 1; i <= iRow; i++)
    {
        for(j = 1; j <= iCol; j++)
        {
            if((j == 1)||(j == iCol)||(i == 1)||(i == iRow))
            {
                printf("*\t");
            }  
            else if(i == j)
            {
                printf("*\t");
            } 
            else
            {
                printf("\t");
            } 
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows :");
    scanf("%d",&iValue1);

     printf("Enter number of column :");
    scanf("%d",&iValue2);

    Pattern(iValue1, iValue2);   //function call

    return 0;
}

File: Assignment_47/program4.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program4.java
// Discription :   use to Display pattern
// Input :         Integer
// Output :        Charecters
// Author :        Shivam Rajendra Kale
// Date :          12/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class pattern
{
    public void Display(int iRow, int iCol)
    {
        int Arr[][] = new int[iRow][iCol];
        int i = 0, j = 0;

        for(i = iRow; i >= 1; i--)
        {
            for(j = 0; j < iCol; j++)
            {
                if(j % 2 == 0)
                {
                    System.out.print("*\t"); 
                }
                else
                {
                    System.out.print("#\t");
                }   
            }
            System.out.println();
        }
    }
}

class program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter number of columns :");
        int iCol = sobj.nextInt();

        pattern pobj = new pattern();

        pobj.Display(iRow,iCol);

        sobj.close();
    }
}